src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -37,7 +37,7 @@ public void execute_personAcceptedByModel_addSuccessful() throws Exception {
 
         CommandResult commandResult = new AddCommand(validPerson).execute(modelStub);
 
-        assertEquals(String.format(AddCommand.MESSAGE_SUCCESS, validPerson), commandResult.getFeedbackToUser());
+        assertEquals(String.format(AddCommand.MESSAGE_SUCCESS), commandResult.getFeedbackToUser());
```

Comment:
Is it better to remove String.format as it is not used here?

Reply:
Thats a good point, should we remove it because it's no longer needed, or do we keep it to maintain consistency across all commands.

Reply:
Yeah, I would also agree to remove String.format as it is no longer used here.